code/modules/mob/dead/new_player/sprite_accessories/wings.dm

```diff
@@ -27,6 +27,24 @@
 	dimension_y = 34
 	locked = TRUE
 
+// Decorative wings
+/datum/sprite_accessory/deco_wings
```

Comment:
you should probably add the insect wings to this as well

copy/paste and just do deco_wings instead of insect_wings

Reply:
Alright, going to do this

Reply:
Done

Reply:
the fairy, bat, etc too, I'd hope :P

Reply:
Pretty much every insect wings that was in the same file